docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md

```diff
@@ -295,149 +293,41 @@ On each node, perform the following as a user with sudo access:
+To enable services to run even when the `yugabyte` user is not logged in, run the following commands:
 
-You can install the backup utility for the backup storage you plan to use, as follows:
-
-- NFS: Install rsync, which YugabyteDB Anywhere uses to perform NFS backups installed during one of the previous steps.
-
-- Amazon S3: Install s3cmd, on which YugabyteDB Anywhere relies to support copying backups to Amazon S3. You have the following installation options:
-
-  - For a regular installation, execute the following:
-
-      ```sh
-      sudo dnf install s3cmd
-      ```
-
-  - For an airgapped installation, copy `/opt/third-party/s3cmd-2.0.1.tar.gz` from the YugabyteDB Anywhere node to the database node, and then extract it into the `/usr/local` directory on the database node, as follows:
-
-      ```sh
-      cd /usr/local
-      sudo tar xvfz path-to-s3cmd-2.0.1.tar.gz
-      sudo ln -s /usr/local/s3cmd-2.0.1/s3cmd /usr/local/bin/s3cmd
-      ```
-
-- Azure Storage: Install azcopy using one of the following options:
-
-  - Download `azcopy_linux_amd64_10.13.0.tar.gz` using the following command:
-
-      ```sh
-      wget https://azcopyvnext.azureedge.net/release20211027/azcopy_linux_amd64_10.13.0.tar.gz
-      ```
-
-  - For airgapped installations, copy `/opt/third-party/azcopy_linux_amd64_10.13.0.tar.gz` from the YugabyteDB Anywhere node, as follows:
-
-      ```sh
-      cd /usr/local
-      sudo tar xfz path-to-azcopy_linux_amd64_10.13.0.tar.gz -C /usr/local/bin azcopy_linux_amd64_10.13.0/azcopy --strip-components 1
-      ```
-
-- Google Cloud Storage: Install gsutil using one of the following options:
-
-  - Download `gsutil_4.60.tar.gz` using the following command:
-
-      ```sh
-      wget https://storage.googleapis.com/pub/gsutil_4.60.tar.gz
-      ```
-
-  - For airgapped installations, copy `/opt/third-party/gsutil_4.60.tar.gz` from the YugabyteDB Anywhere node, as follows:
-
-      ```sh
-      cd /usr/local
-      sudo tar xvfz gsutil_4.60.tar.gz
-      sudo ln -s /usr/local/gsutil/gsutil /usr/local/bin/gsutil
-      ```
-
-## Set crontab permissions
-
-YugabyteDB Anywhere supports performing YugabyteDB liveness checks, log file management, and core file management using cron jobs.
-
-Note that sudo is required to set up this service.
-
-If YugabyteDB Anywhere will be using cron jobs, ensure that the `yugabyte` user is allowed to run crontab:
-
-- If you are using the `cron.allow` file to manage crontab access, add the `yugabyte` user to this file.
-- If you are using the `cron.deny` file, remove the `yugabyte` user from this file.
+```sh
+loginctl enable-linger yugabyte
+vi ~/.bashrc
+export XDG_RUNTIME_DIR=/run/user/$(id -u yugabyte)
+```
 
-If you are not using either file, no changes are required.
+{{< note title="Ulimits on RHEL 8" >}}
 
-<!--
+On RHEL 8 systems, additionally, enter the following commands:
```

Comment:
on RHEL 8 based systems (AlmaLinux 8, RedHat 8, AmazonLinux 2, etc.)

Reply:
And also move this to after the systemd service files definition as a warning/caution